.circleci/config.yml

```diff
@@ -2,19 +2,19 @@ version: 2.1
 
 references:
   repo_path: &repo_path
-      /home/circleci/repo
+               /home/circleci/repo
   ng_workspace_path: &ng_workspace_path
-      /home/circleci/repo/ddp-workspace
+                       /home/circleci/repo/ddp-workspace
   npm_cache_key: &npm_cache_key
-    v1-dependency-npm2-{{ checksum "package.json" }}-{{ checksum "package-lock.json" }}
+                   v1-dependency-npm2-{{ checksum "package.json" }}-{{ checksum "package-lock.json" }}
   npm_cache_restore_key: &npm_cache_restore_key
     keys:
       - *npm_cache_key
   # Note: important that values of guids and keys have matching order!"
   study_keys: &study_keys
-    "basil osteo brain angio mbc testboston mpc prion atcp rarex rgp esc cgc circadia pancan"
+                "basil osteo brain angio mbc testboston mpc prion atcp rarex rgp esc cgc circadia pancan"
   study_guids: &study_guids
-    "basil CMI-OSTEO cmi-brain ANGIO cmi-mbc testboston cmi-mpc PRION atcp rarex RGP cmi-esc cgc circadia cmi-pancan"
+                 "basil CMI-OSTEO cmi-brain ANGIO cmi-mbc testboston cmi-mpc PRION atcp rarex RGP cmi-esc cgc circadia cmi-pancan"
```

Comment:
It's probably OK, but could you undo the whitespace changes here? YAML whitespace matters these make me anxious.